src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -0,0 +1,97 @@
+package net.datafaker.providers.base;
+
+import net.datafaker.providers.entertainment.EntertainmentProviders;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Base64;
+
+import static java.awt.Color.WHITE;
+
+/**
+ * Generates base64 encoded PNG, GIF, JPG and SVG images.
+ *
+ * @since 2.3.0
+ */
+public class Image extends AbstractProvider<EntertainmentProviders> {
+
+    private static final int WIDTH = 256;
+    private static final int HEIGHT = 256;
+    private static final int BOX_SIZE = WIDTH / 8;
+
+    protected Image(EntertainmentProviders faker) {
+        super(faker);
+    }
+
+    public String base64PNG() {
+        return generateBase64Image("png");
+    }
+
+    public String base64JPG() {
+        return generateBase64Image("jpg");
+    }
+
+    public String base64GIF() {
+        return generateBase64Image("gif");
+    }
+
+    public String base64SVG() {
+        return generateBase64SVGImage();
+    }
+
+    private String generateBase64Image(String format) {
+        BufferedImage bufferedImage = new BufferedImage(WIDTH, HEIGHT, BufferedImage.TYPE_INT_RGB);
+        Graphics2D graphics = bufferedImage.createGraphics();
+
+        // Fill the image with white background
+        graphics.setColor(WHITE);
+        graphics.fillRect(0, 0, WIDTH, HEIGHT);
+
+        // Draw random colored boxes
+        for (int y = 0; y < HEIGHT; y += BOX_SIZE) {
+            for (int x = 0; x < WIDTH; x += BOX_SIZE) {
+                int r = faker.random().nextInt(256);
+                int g = faker.random().nextInt(256);
+                int b = faker.random().nextInt(256);
```

Comment:
probably usage of `net.datafaker.service.RandomService#nextRandomBytes` will be much faster since you can generate a number of bytes at once and as I see byte range perfectly suits to your case

Reply:
done!